lib/build/helpers/TaskUtil.js

```diff
@@ -195,6 +195,9 @@ class TaskUtil {
 	 * @property {Function} getSourcePath Get the local File System path of the project's source directory
 	 * @property {Function} getCustomConfiguration Get the project Custom Configuration
 	 * @property {Function} isFrameworkProject Check whether the project is a UI5-Framework project
+	 * @property {Function} getFrameworkName Get the project's frameworkName configuration
```

Comment:
above we have used the syntax `framework project`. Should write here `Get the project's framework name`. Same for the other new methods?

Reply:
Yes, but I would keep the "configuration" to make it clear that it's nothing implicit (like for the isFrameworkProject check)